docs/rfcs/0004-suggested-excluded-apps-tech-spec.md

```diff
@@ -0,0 +1,139 @@
+        -   After running the job, we will create a 24 hour countdown timer to run the job next. Additionally, we'll record the time for this job as a setting, and we'll check this timestamp on app/daemon launch - if more than 24 hours have passed we'll
+    -   Create a dev option to pull a new suggested apps list immediately.
+-   (3-5 points) Is there one or more new apps?
+    -   Check against setting - if we have ever sent a notification for this app before, don't send one (this should be done off the app's domain name or something, not marketing name - that can change)
+    -   Pop an OS-level notification (on Android, use a suggestion channel). The text is intentionally generic, and makes sense if there is one new app or multiple.
+    -   Add new app(s) to "have sent notification" list
+    -   (Create a dev option to clear the "have sent notification" list. Create another dev option to immediately run the daily check job.)
+-   (1 point) Add this notification type to the settings screen that allows per-type opt out (App permissions -> Notifications)
+-   (2 points) Ensure tapping notification leads to correct screen
+
+### Improve existing split tunnel screen (10-16 points total)
+[Figma mocks](https://www.figma.com/file/UZYzma7hlcfE5ke3z8jGbN/App-exclusions-suggestions?type=design&node-id=196-6366&mode=design&t=RL1hdfBQLMS1rKVa-0)
+
+<img src="./images/0004-03.png" width=500 alt="Updated flow for existing App Exclusion screen">
+
+-   (2 points) Run the task on the launch of screen
+    -   Update the "have seen these apps" settings list with all suggested apps that are on the system. (This means that if a new app is seen on the Settings screen before the daily job runs and we send a notification, we never send a notification. This was confirmed by Santiago.)
+-   (1-2 points) Add recommended section (only if 1 more more apps on the system are suggested) - pull any apps in the "have seen them" list
+-   (1 point) Sort recommended section
+-   (1 point) Sort "others" section
+-   (1-2 points) Update sort when something is added or removed
+-   (1-2 points) Ensure search works, shows one section, is alphabetically sorted
+-   (2-5 points) Ensure accessibility is up to date
+-   (1 point) Add metrics for how many suggested (and non-suggested) apps are excluded\
+
+### Day 2
+
+These are important components that we are committed to delivering, but are outside the scope of this epic. They are items that will be done as soon as the epic is closed, but do not block the feature on shipping.
+
+-   Engineering metrics
+    -   Metrics will be created for a successful deep linking, as well as an erroneous deep linking (the app cannot figure out how to resolve the deep link).
+    -   Add performance metrics around load time of the Excluded Apps screen.
+    -   Create a metric for when the daily job was successful, and another one for when it fails.
+    -   A metric should be recorded when the notification is sent
```

Comment:
This was Adrienne's recommendation to try to slim down implementation. I'm in agreement that this is small, but if we're trying to hold to a smallest possible MVP these do seem like the type of thing to get booted to post-launch. Realistically, this will likely go out to users at the same time (unless there is a release branch cut in between the main work and getting this phase two work done).